### PR TITLE
feat: Adding general description of attendee form

### LINKED ISSUE #3189: Wizard: Attendee Form - Add descriptive info
The attendee form in the wizard is missing information where the data is kept. Please add the following:

The attendee form by default collects the name and email of an attendee. You also have the option to collect additional information such as contact data, position, organization and more. This information will be available together with the ticket information in a CSV file in the ticket section of your event.

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #3189 

#### Short description of what this resolves:

- Added `Description` about `Attendee Form`

image

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by kushthedude** [comment]
@mariobehling Please have a look, If this looks alright.

**Comment by kushthedude** [comment]
@niranjan94 @CosmicCoder96 Please Review

File: app/templates/components/forms/wizard/attendee-step.hbs [hunk 1]
```diff
@@ -9,15 +9,20 @@
               {{input type='checkbox' checked=data.event.isTicketFormEnabled}}
               <label class="weight-300" style="font-size: large">
                 {{if data.event.isTicketFormEnabled (t 'Turn off') (t 'Turn on')}}
-                {{t 'Ticket Form'}}
```
Comment (shreyanshdwivedi) [inline app/templates/components/forms/wizard/attendee-step.hbs]: @kushthedude a query. Why was it `Ticket Form` earlier? I guess I missed these changes.

Comment (kushthedude) [reply app/templates/components/forms/wizard/attendee-step.hbs]: I don't know, Why was it named `Ticket Form`, But still we should correct our mistakes .

**Comment by kushthedude** [comment]
@CosmicCoder96 Please REview

**Pull request merged**